CHANGELOG.md

```diff
@@ -11,6 +19,9 @@
 - Bump Cocoa SDK from v8.8.0 to v8.18.0 ([#526](https://github.com/getsentry/sentry-capacitor/pull/526))
   - [changelog](https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#8180)
   - [diff](https://github.com/getsentry/sentry-cocoa/compare/8.8.0...8.18.0)
+- Bump JavaScript SDK from v7.81.1 to v7.93.0 ([#529](https://github.com/getsentry/sentry-capacitor/pull/529))
+  - [changelog](https://github.com/getsentry/sentry-javascript/blob/develop/CHANGELOG.md#7930)
+  - [diff](https://github.com/getsentry/sentry-javascript/compare/7.81.1...7.93.0)
```

Comment:
```suggestion
```

Reply:
This entry changes the 0.15.1 changelog, we should remove it, and only keep the changes in the unreleased.